### ISSUE TITLE
Investigate a11y need for setting visibility along with display to hide something

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly#5145]
In a lot of/most places, we use the combination of `display: none; visibility: hidden;` to hide something and `display: [whatever]; visibility: visble;` to show it again. I believe that was the result of a recommendation made for JAWS in
* this PR comment - https://github.com/patternfly/patternfly/issues/915#issuecomment-437157977
  * which links to this article - https://medium.engineering/five-goofy-things-medium-did-that-break-accessibility-3bc804ae818d#ff06
    * which references https://www.a11yproject.com/posts/how-to-hide-content/ with the technique, but that article no longer says to use `display` and `visibility` to accesibly hide something in JAWS.
      * the a11yproject.com article does reference an older [scott o'hara article](https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html) and a newer [kitty giraudel](https://kittygiraudel.com/2021/02/17/hiding-content-responsibly/) article, and both say to use `display` alone with no special mention of `display` and `visibility` together for JAWS, that I could find anyways

cc @jessiehuff @thatblindgeye wdyt?
Need to double check @mcoker recommendation: that `display:none` is all that's needed for JAWS/VO. If it's checks out, then update the code to clean up styles for hiding content.

**Comment by mcoker** [comment]
@thatblindgeye fwiw, I found this article from 2015 when it was removed from html5 boilerplate, and it includes the versions of JAWS that they tested - https://github.com/h5bp/html5-boilerplate/issues/1663

**Comment by thatblindgeye** [comment]
The fact that it looks like the last version of JAWS that this bug was present in was ~13 versions ago (on IE 11 + Windows 7 to boot), that makes me feel more confident about making this update!

I still want to try testing things out on a more current OS + browser combo just to be sure, though I can't imagine (🤞🏼 ) there'd be any issues.

**Comment by thatblindgeye** [comment]
Can at least confirm that JAWS + Chrome/Firefox I was able to confirm that only using `display: none` (using the HTML/CSS from that last link shared) does seem to work 🎉 In Firefox I had to update the `<div>` to an `<a>` (so 2 links) from the HTML supplied in that link you shared, and had to rely on tabbing as JAWS doesn't seem to work well for me in Firefox. Chrome I was able to use some of JAWS built in navigation and keep the example HTMl as-is (1 link + 1 div). Also tested VO + Chrome/Safari/Firefox and NVDA + Chrome/Firefox, and all sounds good there too.

Also tested VO + Chrome/Safari/Firefox, JAWS + Chrome/Firefox, and NVDA + Chrome/Firefox to double check that elements hidden with `display: none` weren't focusable since that was mentioned during pre-planning, and things are good there as well!

Was using either the latest versions or close to latest versions of the screen readers + browsers in my testing.
